bigtable/bigtable.go

```diff
@@ -697,9 +699,12 @@ func makeFullReadStats(reqStats *btpb.RequestStats) FullReadStats {
 			CellsReturnedCount: readStats.CellsReturnedCount,
 			CellsSeenCount:     readStats.CellsSeenCount,
 			RowsReturnedCount:  readStats.RowsReturnedCount,
-			RowsSeenCount:      readStats.RowsSeenCount},
+			RowsSeenCount:      readStats.RowsSeenCount,
```

Comment:
Whoops, format on save. I can revert these parts if we prefer to reduce out-of-context lines.